app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import NotWearingMaskError
+from app.errors import VaccineError
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
+
+    friends_has_access = True
```

Comment:
This variable is redundant. Remove it in all places
Because it is always True

Reply:
done